### PR TITLE
Code refactoring

### PR DESCRIPTION
**Pull request by damianszczepanik**
- Facts -> Fact
 - Sections -> Section
 - reused formatter
 - code formatting

### CODE REVIEW

**Comment by damianszczepanik** [comment]
First step (small refactoring) which helps me to provide new functionalities.

**Comment by casz** [comment]
I suspect it's done with reformat code from a code editor, generally white spaces and plural naming of classes is what I would call code smell.

**Comment by SrhebbarMicrosoft** [comment]
Please resolve the conflicts.

**Comment by casz** [comment]
As documented by [`card-reference`](https://docs.microsoft.com/en-us/outlook/actionable-messages/card-reference) your breaking naming convention set out by Microsoft 👎

File: src/main/java/jenkins/plugins/office365connector/model/Section.java [hunk 1]
```diff
@@ -17,75 +17,63 @@
 import java.util.List;
 
 /**
- *
  * @author srhebbar
  */
-public class Sections
-{
+public class Section {
```
Comment (casz) [inline src/main/java/jenkins/plugins/office365connector/model/Section.java]: Revert rename

File: src/main/java/jenkins/plugins/office365connector/model/Fact.java [hunk 2]
```diff
@@ -17,46 +17,40 @@
 import java.util.List;
 
 /**
- *
  * @author srhebbar
  */
-public class Facts
-{
+public class Fact {
```
Comment (casz) [inline src/main/java/jenkins/plugins/office365connector/model/Fact.java]: Revert rename

File: src/main/java/jenkins/plugins/office365connector/model/Section.java [hunk 3]
```diff
@@ -17,75 +17,63 @@
...
     private boolean markdown;
 
-    private List<Facts> facts;
+    private List<Fact> facts;
```
Comment (casz) [inline src/main/java/jenkins/plugins/office365connector/model/Section.java]: Discard last review... as this is the json based name

File: src/main/java/jenkins/plugins/office365connector/model/Card.java [hunk 4]
```diff
@@ -17,61 +17,53 @@
...
+
     private String summary;
-    
+
     private String themeColor;
-    
-    private List<Sections> sections;
-    
+
+    private List<Section> sections;
```
Comment (casz) [inline src/main/java/jenkins/plugins/office365connector/model/Card.java]: Discard last review... as this is the json based name

**Comment by damianszczepanik** [comment]
Conflicts resolved

**Pull request merged**